tests/v2/test_waku_store.nim

```diff
@@ -88,6 +88,7 @@ procSuite "Waku Store":
     listenSwitch.mount(proto)
 
     await subscriptions.notify("foo", msg)
+    await sleepAsync(1.millis)  # Sleep a millisecond to ensure messages are stored chronologically
```

Comment:
Shouldn't this be on second resolution if we are using epoch?

Reply:
The epoch time supports subsecond resolution (up to a microsecond). 1 millisecond is therefore plenty to force floating timestamps to increase.